pesa_migration/pesa_set_options.sql

```diff
@@ -6,6 +6,9 @@ update ##new_db##.or_options set option_value = 'migration' where (option_id = '
 update ##new_db##.or_options set option_value = 'n' where (option_id = '1426010286'); /* disable ethics module */
 update ##new_db##.or_options set option_value = 'y' where (option_id = '1426010298'); /* allow modification of reminder mails */
 update ##new_db##.or_options set option_value = 'y' where (option_id = '1426010305'); /* allow modification of sender mail */
+update ##new_db##.or_options set option_value = 10  where (option_id = '1426012101'); /* set max session duration hour default */
```

Comment:
this could be 23 hours. No reason to restrict it further. (23 just because I assume the logic of the code assumes a session to be shorter than a day).

Reply:
I set it to 24 hours.